locksmith/src/routes/index.ts

```diff
@@ -31,6 +31,7 @@ import emailRouter from './v2/email'
 import checkoutConfigRouter from './v2/checkoutConfigs'
 import config from '../config/config'
 import stripeRouter from './v2/stripe'
+import settingsRouter from './v2/settings'
```

Comment:
Let's make it more specific to the lock:

```suggestion
import lockSettingsRouter from './v2/lock-settings'
```